src/components/shared/slider/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {Flex, Slider as SliderNB, Text, View} from 'native-base';
+import {
+  Control,
+  Controller,
+  FieldValues,
+  Path,
+  UnPackAsyncDefaultValues,
+} from 'react-hook-form';
+import CustomTooltip from '../../tooltip';
+import styles from './styles';
+
+interface Props<TFormValues extends FieldValues> {
+  name: Path<UnPackAsyncDefaultValues<TFormValues>>;
+  label?: string;
+  control: Control<TFormValues>;
+  minLimit?: number;
+  maxLimit?: number;
+}
+
+const Slider = <
+  TFormValues extends {
+    [key: string]: string;
+  },
+>({
+  name,
+  label,
+  control,
+  minLimit = 0,
+  maxLimit = 100,
+}: Props<TFormValues>) => {
+  return (
+    <View w="75%">
+      <View style={styles.tooltipContainer}>
+        <Text>{label}</Text>
+        <CustomTooltip
+          title="Skill Functionality"
+          description="The skill level will help us to leverage the draft better between both teams"
+        />
```

Comment:
Add a showTooltip customProp false by default in case we want to use this component with no tooltip
```suggestion
        {showTooltip && <CustomTooltip
          title="Skill Functionality"
          description="The skill level will help us to leverage the draft better between both teams"
        />}
```